src/core/server/integration_tests/node/migrator.test.ts

```diff
@@ -42,7 +44,7 @@ describe('migrator-only node', () => {
           '--no-optimizer',
           '--no-base-path',
           '--no-watch',
-          '--oss',
+          isFipsEnabled ? '--xpack.security.experimental.fipsMode.enabled=true' : '--oss',
```

Comment:
I’m surprised we’re still testing OSS stuff (maybe we could save some CI costs… just saying) 🙈

Reply:
🤷‍♂️ They were runnin OSS when I found them!

Reply:
As I understand it, the --oss flag actually helps these integration tests start up a bit faster since we are excluding loading any xpack plug-ins.